presto-main/src/test/java/com/facebook/presto/sql/query/TestSubqueries.java

```diff
@@ -86,16 +86,16 @@ public void testCorrelatedExistsSubqueriesWithOrPredicateAndNull()
     }
 
     @Test
-    public void testUnsupportedSubqueriesWithCoercions()
```

Comment:
I think limit here was incidental and we still want this test for unsupported subqueries to ensure they throw proper errors.

Reply:
then I will add a seperate tc with `group by a limit 1` to guard this sanity test